tests/fv3_conf/fv3_slurm.IN_noaacloud

```diff
@@ -12,11 +12,14 @@ set -eux
 echo -n " $( date +%s )," >  job_timestamp.txt
 
 set +x
-MACHINE_ID=noaacloud
-module use $( pwd -P )
-module use /contrib/EPIC/spack-stack/spack-stack-1.3.0/envs/unified-dev/install/modulefiles/Core
-module load stack-intel/2021.3.0  stack-intel-oneapi-mpi/2021.3.0
-module load ufs-weather-model-env/unified-dev
+export MACHINE_ID=noaacloud
+source ./module-setup.sh
+if [[ ! -d modulefiles ]]; then
+  mkdir modulefiles
```

Comment:
why are these changes needed? i prefer to not modify anything in the top level tests directory if we don't have to.